terraform/fleet/sg.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_security_group" "ae-nodes" {
+    name = "ae-${var.env}-nodes-terraform" #postfix with terraform to avoid issues with current setup
+    lifecycle {
+        create_before_destroy = true
+    }
+}
+
+resource "aws_security_group_rule" "allow_all_internal" {
+    type = "ingress"
+    from_port = 0
+    to_port = 65535
+    protocol = "TCP"
+
+    security_group_id = "${aws_security_group.ae-nodes.id}"
+    source_security_group_id = "${aws_security_group.ae-nodes.id}"
+}
+
+resource "aws_security_group_rule" "allow_outgoing-node" {
+    type = "egress"
+    from_port = 0
+    to_port = 65535
+    protocol = "TCP"
+    cidr_blocks     = ["0.0.0.0/0"]
+    security_group_id = "${aws_security_group.ae-nodes.id}"
+}
+
+resource "aws_security_group" "ae-nodes-management" {
+    name = "ae-${var.env}-management-terraform"
+    lifecycle {
+        create_before_destroy = true
+    }
+}
+
+resource "aws_security_group_rule" "allow_ssh" {
+    type = "ingress"
+    from_port = 22
+    to_port = 22
+    protocol = "TCP"
+    cidr_blocks     = ["0.0.0.0/0"]
+    security_group_id = "${aws_security_group.ae-nodes-management.id}"
+}
+
+resource "aws_security_group_rule" "allow_epoch_1" {
+    type = "ingress"
+    from_port = 3013
```

Comment:
I know this rule is currently under management SG in our setups, but it's oldish, let's move both node ports to `ae-nodes` SG?

Reply:
there are 2 SG groups, one with just ssh second with 30*

Reply:
do not understatd